examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in

```diff
@@ -0,0 +1,17 @@
+File f_feats_train = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
+
+#![create_features]
+Features feats = create_features(f_feats_train)
+#![create_features]
+
+#![set_parameters]
+Transformer fa = create_transformer("LaplacianEigenmaps", target_dim=1, k=20, tau=20)
```

Comment:
tau needs to be a float.
Please test those things before you send a PR

```
make build_cpp_meta_examples
ctest -R laplacian_eigen_maps
```

Reply:
Make sure to read the error msgs from the CI. And also *please test meta examples locally before you send them*. This doesnt execute.

https://dev.azure.com/shogunml/shogun/_build/results?buildId=3927&view=logs&j=014b6934-7273-5d37-4c53-a79dadc49884&t=53126599-58a0-5542-c2f0-17b2aa146982&l=345

Reply:
Yes, sorry. I just tested one of them and then pushed all. I'll be more careful.

Reply:
no worries, I think it should be very little work to test all compared to one.